srtcore/handshake.h

```diff
@@ -113,26 +115,19 @@ typedef Bits<15, 0> SRT_HS_LATENCY_SND;
 typedef Bits<15, 0> SRT_HS_LATENCY_LEG;
 
 
-// XXX These structures are currently unused. The code can be changed
-// so that these are used instead of manual tailoring of the messages.
 struct SrtHandshakeExtension
```

Comment:
This actually removes the existing - still unused - `SrtHandshakeExtension` type and replaces it with something useful.